resources/splunk_install.rb

```diff
@@ -146,10 +153,9 @@ def kernel_string
   action :install do
     super()
 
-    converge_if_changed :version, :build do
-      poise_archive package_path do
-        destination install_dir
-      end
+    poise_archive package_path do
+      destination install_dir
+      only_if { changed? :version, :build }
```

Comment:
this is functionally identical right?  was this part changed just for consistency with other sections?